docs/production.rst

```diff
@@ -141,6 +141,28 @@ want to change the password as soon as you log in):
 * **username**: admin
 * **password**: adminadmin
 
+.. _email:
+
+Email
+-----
+
+The application is configured to use Django's console email backend by default, which causes messages intended to be sent via email to be written to ``stdout``.
+
+In order to use a real email server, another backend listed in `Django's documentation`_ can be configured instead. Parameters for the backend are set as variables in ``settings.py``. For example, the following allows an SMTP server at ``smtp.example.com`` to be used::
+
+   Email_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+   ENABLE_EMAIL = True
+   EMAIL_HOST = 'smtp.example.com'
+   EMAIL_PORT = 587
+   EMAIL_HOST_USER = 'wger@example.com'
+   EMAIL_HOST_PASSWORD = 'example_password'
+   EMAIL_USE_TLS = True
+   EMAIL_USE_SSL = False
+   DEFAULT_FROM_EMAIL = 'wger Workout Manager <wger@example.com>'
+
+Email settings can be tested using the password reset function in the login page, provided an account with an email address has been created.
```

Comment:
there is a management command in django for this, `python manage.py sendtestemail my@email.com`, that's probably a cleaner way to test the configuration

Reply:
Thanks for pointing that out, that's a much better option.